Swap First Compute Job and Interactive Compute chapters

caalo opened the pull request:
In teaching Cluster 101 workshop, it has been easier for students to work with the compute node in interactive mode before submitting bash computing scripts. This PR reflects the change in teaching order.

avahoffman commented:
<!-- **/*.md-code-coverage-assistant -->
**Overall readability score:** 52.39 (🟢 +0.03)

File | Readability
--- | ---
[README.md](https://github.com/fhdsl/FH_Cluster_101/blob/673531209ca9c9f15c3fd1b2bad1ef9aca95640b/README.md "README.md") | 69.7 (🟢 +2.13)


<details>
  <summary>View detailed metrics</summary>

🟢 - Shows an _increase_ in readability
🔴 - Shows a _decrease_ in readability

File | Readability | FRE | GF | ARI | CLI | DCRS
--- | --- | --- | --- | --- | --- | ---
[README.md](https://github.com/fhdsl/FH_Cluster_101/blob/673531209ca9c9f15c3fd1b2bad1ef9aca95640b/README.md "README.md") | 69.7 | 66.44 | 8.76 | 9.2 | 10.54 | 7.69
&nbsp; | 🟢 +2.13 | 🟢 +6.64 | 🔴 -0.67 | 🟢 +0.3 | 🟢 +0.87 | 🟢 +0.16


Averages:

&nbsp; | Readability | FRE | GF | ARI | CLI | DCRS
--- | --- | --- | --- | --- | --- | ---
Average | 52.39 | 46.69 | 11.19 | 13.38 | 12.73 | 7.91
&nbsp; | 🟢 +0.03 | 🟢 +0.09 | 🔴 -0.01 | 🟢 +0 | 🟢 +0.01 | 🟢 +0


<details>
  <summary>View metric targets</summary>

Metric | Range | Ideal score
--- | --- | ---
Flesch Reading Ease | 100 (very easy read) to 0 (extremely difficult read) | 60
Gunning Fog | 6 (very easy read) to 17 (extremely difficult read) | 8 or less
Auto. Read. Index | 6 (very easy read) to 14 (extremely difficult read) | 8 or less
Coleman Liau Index | 6 (very easy read) to 17 (extremely difficult read) | 8 or less
Dale-Chall Readability | 4.9 (very easy read) to 9.9 (extremely difficult read) | 6.9 or less

</details>

</details>

caalo commented:
Based on feedback from Ava and SciComp, added best practices of using Job Scripts over Interactive Mode: the reason we start with Interactive Mode is that it is a good learning tool and for occasional tests, but ultimately on should be writing Job Scripts for any substantial compute job on the cluster.

Also renamed course to Fred Hutch Cluster Computing.
